forte/processors/data_augment/algorithms/distribution_replacement_op.py

```diff
@@ -127,14 +127,61 @@ def cofigure_sampler(self) -> None:
                 },
             )
         except KeyError as error:
-            print("Could not configure Sampler: " + repr(error))
+            raise Exception from error
 
     @classmethod
     def default_configs(cls):
+        r"""
+        Returns:
+            A dictionary with the default config for this processor.
+        Following are the keys for this dictionary:
+
+            - `prob`:
+                The probability of whether to replace the
+                input, it should fall in `[0, 1]`. Default
+                value is 0.1
+
+            - `sampler_data`:
+                A dictionary representing the configurations
+                required to create the required sampler.
+
+                - type:
+                    The type of sampler to be used (pass the
+                    path of the class which defines the required sampler)
+
+                - kwargs:
+                    This dictionary contains the data that is to be
+                    fed to the required sampler. 2 possible values are
+                    `sampler_data` and `data_path`.If both parameters are passed,
+                    the data read from the file pointed to by `data_path` will be
+                    considered.
+
+                    - `sampler_data`:
+                        Raw input to the sampler, This will be passed as the
+                        `sampler_data`
+                        config to the required sampler.
+
+                    - `data_path`:
+                        The path to the file that contains the
+                        the input that will be given to the sampler. For example,
+                        when using `UniformSampler`, `data_path` will point to a file
+                        (or `URl`) containing a list of values to be used as
+                        `sampler_data` in `UniformSampler`.
+
+                .. code-block:: python
+
+                    {
+                        "type": "forte.processors.data_augment.algorithms.sampler.UniformSampler",
+                        "kwargs":{
```

Comment:
1. you remove the `kwargs` from implementaion but not the docstring here.
2. This is actually the correct place to use `kwargs` since we are constructing a `UniformSampler` using the `kwargs` provided. We can drop `no_typecheck` and keep using `kwargs` here.